CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## v0.4.1 (2025-02-05)
```

Comment:
I mention in a previous comment to link to the diff, but I'm still not seeing these changes in this PR or the prior changes since `v0.4.0`

https://github.com/change/linguist/compare/v0.4.0...main

```
* Improve documentation and add more examples
* Enhance CI pipeline to include more Elixir and OTP versions
* Add support for new locale formats
* Remove deprecated functions and clean up codebase
```